metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/service/ProxiesService.java

```diff
@@ -395,6 +394,52 @@ public RecordsResponse getListOfFileContentsFromPluginExecution(MetisUserView me
     return new RecordsResponse(records);
   }
 
+  /**
+   * Get a list with record contents from the external resource based on a workflow execution and the predecessor
+   * of the given {@link PluginType}.
+   *
+   * @param metisUserView the user wishing to perform this operation
+   * @param workflowExecutionId the execution identifier of the workflow
+   * @param pluginType the {@link ExecutablePluginType} that is to be located inside the workflow
+   * @param ecloudIds the list of ecloud IDs of the records we wish to obtain
+   * @return the list of records from the external resource
+   * @throws GenericMetisException can be one of:
+   * <ul>
+   * <li>{@link MCSException} if an error occurred while retrieving the records from the external
```

Comment:
Should be `eu.europeana.metis.exception.ExternalTaskException`.
(in develop this should have already been changed for the other methods in this class)

Reply:
Updated